autoload/lsp/lsp.vim

```diff
@@ -43,36 +43,16 @@ def LspInitOnce()
   lspInitializedOnce = true
 enddef
 
-# Returns the LSP server for the a specific filetype. Returns an empty dict if
-# the server is not found.
-def LspGetServer(bnr: number, ftype: string): dict<any>
+# Returns the LSP servers for the a specific filetype. Returns an empty list if
+# the servers is not found.
+def LspGetServers(bnr: number, ftype: string): list<dict<any>>
   if ftypeServerMap->has_key(ftype)
-    var lspservers = ftypeServerMap[ftype]
-
-    # The best language server for the current buffer
-    var bestMatch: dict<any> = {}
-    var bestScore: number = -1
-
-    var bufDir = bnr->bufname()->fnamemodify(':p:h')
+    return ftypeServerMap[ftype]
 
-    for lspserver in lspservers
-      var score: number = 0
-      if !lspserver.rootSearchFiles->empty()
-        # The score is calculated by how deep the workspace root dir is, the
-        # deeper the better.
-        var path = util.FindNearestRootDir(bufDir, lspserver.rootSearchFiles)
-        score = path->strcharlen()
-      endif
-      if score > bestScore
-        bestMatch = lspserver
-        bestScore = score
-      endif
-    endfor
-
-    return bestMatch
+    # TODO score based inclusion/exclusion
```

Comment:
Currently the code implemented in #228 is being broken by this PR.

What should we do about it?